packages/angular/projects/ui-angular/src/lib/primitives-poc/password-field/password-field-component/password-field.component.html

```diff
@@ -0,0 +1,59 @@
+<div class="amplify-flex amplify-field amplify-textfield">
+  <!-- Label -->
+  <label [for]="inputId" *ngIf="!labelHidden">
+    <ng-content select="[amplify-label]"> </ng-content>
+  </label>
+
+  <!-- Description Text -->
+  <ng-content select="[amplify-description-text]"></ng-content>
+
+  <!-- Actual Input -->
+  <div
+    class="amplify-flex amplify-field-group amplify-field-group--horizontal"
+    data-orientation="horizontal"
+  >
+    <div
+      class="
+        amplify-field-group__field-wrapper
+        amplify-field-group__field-wrapper--horizontal
+      "
+      data-orientation="horizontal"
+    >
+      <ng-content select="[amplify-password-input]"></ng-content>
+    </div>
+    <div class="amplify-field-group__outer-end">
+      <button
+        aria-label="Show password"
+        class="
+          amplify-button
+          amplify-field-group__control
+          amplify-button--small
+          amplify-field__show-password
+        "
+        data-fullwidth="false"
+        data-size="small"
+        type="button"
+        aria-checked="true"
+        role="switch"
+        (click)="togglePassword()"
+      >
+        <span class="amplify-visually-hidden" aria-live="polite">{{
+          passwordIconStr
+        }}</span
+        ><span class="amplify-icon" style="width: 1em; height: 1em"
+          ><svg
+            width="24"
+            height="24"
+            viewBox="0 0 24 24"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+          >
```

Comment:
Formatting is still off;

```suggestion
        <span class="amplify-visually-hidden" aria-live="polite">
          {{  passwordIconStr  }}
        </span>
        <span class="amplify-icon" style="width: 1em; height: 1em">
          <svg
            width="24"
            height="24"
            viewBox="0 0 24 24"
            fill="none"
            xmlns="http://www.w3.org/2000/svg"
          >
```